bitnami/postgresql/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 15.5.22
+version: 15.5.23
```

Please bump a minor version since new values where added.
```suggestion
version: 15.6.0
```